doc/qbittorrent.1

```diff
@@ -1,41 +1,236 @@
+If this category does not exist, it will be automatically created.
+.TP
+\f[B]\f[CB]--first-and-last\f[B]\f[R]
+Download first and last pieces first.
+.TP
+\f[B]\f[CB]--save-path=\f[B]\f[R]\f[I]PATH\f[R]
+Save the torrent(s) to the specified \f[I]PATH\f[R].
+TODO: if relative, is it relative to qbit\[cq]s working dir?
+.TP
+\f[B]\f[CB]--sequential\f[B]\f[R]
+Download pieces in sequential order.
+.TP
+\f[B]\f[CB]--skip-dialog=\f[B]\f[R]\f[I]TOGGLE\f[R]
+\f[I]TOGGLE\f[R] is one of \f[B]true\f[R] or \f[B]false\f[R].
+If \f[B]true\f[R], the \[lq]Add New Torrent\[rq] dialog will not be
+shown.
+Defaults to \f[B]false\f[R].
+TODO: again, why does this require a true/false parameter?
+.TP
+\f[B]\f[CB]--skip-hash-check\f[B]\f[R]
+By default, if a torrent\[cq]s files exist in the destination save path,
+a recheck is initiated.
+This option prevents that from happening, which is not recommended in
+general.
+Torrents may still be force-rechecked after they are added, as usual.
+If the torrent\[cq]s files do not exist in the destination save path,
+this option has no effect.
+.SH ENVIRONMENT
 .PP
-\f[B]\f[CB]--help\f[B]\f[R] Prints the command line options.
+\f[I]OPTIONS\f[R] may be supplied via environment variables with
+equivalent names.
 .PP
-\f[B]\f[CB]--version\f[B]\f[R] Prints qbittorrent program version
-number.
+For an option named \f[B]parameter-name\f[R], the corresponding
+environment variable name is \f[B]QBT_PARAMETER_NAME\f[R].
+In other words, the parameter name is converted to upper-case, any
+hyphens (\f[B]-\f[R]) are replaced by underscores (\f[B]_\f[R]), and
+\f[B]QBT_\f[R] is prepended to it.
 .PP
-\f[B]\f[CB]--no-splash\f[B]\f[R] Disables splash screen on startup.
+To pass flag values, set the variable to \f[B]1\f[R] or \f[B]TRUE\f[R].
+For example, to disable the splash screen: \f[B]QBT_NO_SPLASH=1\f[R]
+.SH FILES
 .PP
-\f[B]\f[CB]--webui-port=x\f[B]\f[R] Changes Web UI port to x (default:
-8080).
+qBittorrent complies with the Freedesktop XDG Base Directory
+specification.
+TODO: not always, see \[lq]data\[rq] folder.
+Typically, \f[B]$XDG_CONFIG_HOME\f[R] is \f[I]\[ti]/.config\f[R] and
+\f[B]$XDG_DATA_HOME\f[R] is \f[I]\[ti]/.local/share\f[R].
+.PP
+TODO: did I miss any?
+.TP
+\f[I]\f[BI]$XDG_CONFIG_HOME\f[I]/qBittorrent/qBittorrent.conf\f[R]
+Main configuration file.
+.TP
+\f[I]\f[BI]$XDG_CONFIG_HOME\f[I]/qBittorrent/qBittorrent-data.conf\f[R]
+This is where all-time statistics are saved.
+.TP
+\f[I]\f[BI]$XDG_CONFIG_HOME\f[I]/qBittorrent/rss/feeds.json\f[R]
+RSS feeds configuration.
+.TP
+\f[I]\f[BI]$XDG_DATA_HOME\f[I]/data/qBittorrent/BT_backup/\f[R]
+This directory contains the \f[I].fastresume\f[R] files, which is the
+mechanism qBittorrent uses to keep track of the state of each added
+torrent across restarts.
+In addition, a copy of each of the added torrents\[cq] meta-files is
+kept here, with the torrents\[cq] hashes as the file names.
+.TP
+\f[I]\f[BI]$XDG_DATA_HOME\f[I]/data/qBittorrent/GeoIP/GeoLite2-Country.mmdb\f[R]
```

Comment:
/GeoDB/dbip-country-lite.mmdb

Reply:
You only need to review the `.md` file, this one is auto-generated from the `.md` file.